### PR TITLE
Pricing rules

### PR DESCRIPTION
**Pull request by henrikingo**
PricingPage additions:
* Pay 12 month upfront
* Display text if engineering dept is unusually small
* Display text if company is 1000 employees or more

Note that last cómmit is unrelated, but was bothering me, so fixing it while here.

### CODE REVIEW

**Comment by henrikingo** [comment]
@mfleming Mostly asking for review whether you agree with the pricing proposed?

**Comment by mfleming** [comment]
Can you summarise the changes?

- If the engineering dept is < 25% ask them to call us
- Provide 20% discount if paying annually

Anything else?

Also, the "Save 20% by paying up front" calculation for the saving doesn't work right

image

In general you shouldn't sneak in unrelated changes so please make the dashboard change a separate PR.

File: frontend/src/components/PricingPage.jsx [hunk 1]
```diff
@@ -93,9 +108,123 @@ export const PricingPage = () => {
...
+
+    //console.debug("If Engineering dept isn't 25-50% of total employees, give custom price after talking to them.")
+    if (eng>0 && total > 0
+        && eng / total<0.25
+        && total < 1000){
+      console.debug("Small engineering dept -> custom price notice")
+      return (
+          <>
+          <div className="nyrkio-pricing-note nyrkio-pricing-engineering-small">
+            <p>Note: Our list price may not be perfect for your organization. Please <a href="">contact us</a> so we can discuss a more suitable pricing level. We want to ensure Nyrkiö is a good fit for projects large and small.</p>
```
Comment (mfleming) [inline frontend/src/components/PricingPage.jsx:195]: These hrefs should point somewhere.

**Comment by henrikingo** [comment]
> Anything else?

If bigger than 1000 employees, call us

> Also, the "Save 20% by paying up front" calculation for the saving doesn't work right

The quoted number is 20% off the Enterprise price. Maybe it's confusing? Would it help if we just add "(Enterprise level)" at the end of that sentence?

> In general you shouldn't sneak in unrelated changes so please make the dashboard change a separate PR.

Sure, if you prefer. Last PR you didn't review at all, so I misinterpreted level of process vs mad startup.

> These hrefs should point somewhere.

Sigh. :man_facepalming: 

Anyway, I was going with email address for now. At some point we may want to offer a calendly link or phone number... So they could literally "Call us". WDYT?

**Comment by henrikingo** [comment]
@mfleming Note that all of my PRs are updated and ready to go, but I didn't merge any of them as you had commented on this one and the others were spun off from this one too.

**Review by mfleming — APPROVED** [review]

**Pull request merged**